test/custom-operations-test.js

```diff
@@ -1,24 +1,21 @@
-import * as assert from "assert";
-import sift from "..";
+import * as assert from 'assert';
+import sift from '..';
 
-describe(__filename + "#", () => {
-  it("can add a custom $mod operation", function() {
-    var filter = sift(
-      { $mod2: 2 },
+describe('custom operations', () => {
```

Comment:
Could we add a custom async filter test?

Reply:
7687686b8044332e3c3cad6f435d4b1bbc16b577